test/unit/alert_test.rb

```diff
@@ -12,12 +12,8 @@ class AlertTest < ActiveSupport::TestCase
   should validate_numericality_of(:utilization)
 
   def test_by_level
-    FactoryBot.create(:limit_alert, level: 50)
-    FactoryBot.create(:limit_alert, level: 80)
-
-    alerts = Alert.by_level(80)
-    assert_kind_of Alert::ActiveRecord_Relation, alerts
-    assert_equal 1, alerts.count
+    alert = FactoryBot.create(:limit_alert, level: 80)
```

Comment:
Wait, why did you remove the level 50 one? Does not that change the meaning of the test?

Reply:
I think No, because we are not even using it anywhere we have just created it, It was useful when we are checking count.

Reply:
Cleared doubt with @akostadinov and sorted the PR. @hallelujah